File: 18-Problem-RepeatSixteenAgain/18-Problem-RepeatSixteenAgain.swift
// Repeat problem 16 again to work rote memory of basic syntax.

let NINT: Int = Int.random(in: 1..<10);
let FLOTE: Float = 32.32;
let CHR: Character = "c";
let STR: String = "String";
let BOO: Bool = true;

func incrementNint(num: Int) -> Int {
    var local_num: Int = num;
    if num > 5 {
        var adder: Int = Int.random(in: 1..<10);
        var loops: Int = Int.random(in: 1..<10);

        while loops > 0 {
            local_num+=adder;
            adder+=1;
            loops-=1;

        }
    }
  return local_num;
}

print("NINT: \(NINT) FLOTE: \(FLOTE) CHR: \(CHR) STR: \(STR) BOO: \(BOO)");

print("incrementNint(NINT): \(incrementNint(num: NINT))");

File: 18-Problem-RepeatSixteenAgain/18-Problem-RepeatSixteenAgain-TS/18-Problem-RepeatSixteenAgain.js
// Repeat problem 16 again to work rote memory of basic syntax.
var NINT = Math.floor(Math.random() * (10 - 1) + 1);
var FLOTE = 32.32;
var CHR = 'c';
var STR = "String";
var BOO = true;
var incrementNint = function (num) {
    if (num > 5) {
        var adder = Math.floor(Math.random() * (10 - 1) + 1);
        var loops = Math.floor(Math.random() * (10 - 1) + 1);
        while (loops > 0) {
            num += adder;
            adder++;
            loops--;
        }
    }
    return num;
};
console.log("NINT: ".concat(NINT, " FLOTE: ").concat(FLOTE, " CHR: ").concat(CHR, " STR: ").concat(STR, " BOO: ").concat(BOO));
console.log("incrementNint(NINT): ".concat(incrementNint(NINT)));
